calculators/gas_king_rating.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+$GLOBALS['rating_calculators']['_mignoubou_gas_king'] = [
+    'description' => '
+        by <a href="/player/22267">PewPewPew</a>: gas king rating v2
+        only objective: gas as much and as soon as you can, but do not lose !
+        you start with 2000 rating
+        you lose 10 rating per minute for each gas not taken (without compression) (only when winning)
+        you win 1, 2, 3, 4 rating per minute for gas 1, 2, 3, 4 (without compression)
```

Comment:
it is not clear what compression is. You maybe want to describe it. Up to you.

Reply:
done